packages/medusa/src/services/__tests__/shipping-option.js

```diff
@@ -147,21 +153,32 @@ describe("ShippingOptionService", () => {
           provider_data: "true",
         },
         price_type: "flat_rate",
-        amount: 100,
+        amount: 200,
       })
     })
 
+    it("throws on flat rate but no amount", async () => {
+      try {
+        await optionService.update(IdMap.getId("flat-rate-no-amount"), {
```

Comment:
question: Why are we expecting an error when we are passing the amount here?

Reply:
Nice catch. Updated the test :)